run-cnf-suites.sh

```diff
@@ -53,5 +53,18 @@ done
 
 FOCUS=${FOCUS%?}  # strip the trailing "|" from the concatenation
 
+if [ "$VERIFY_CNF_FEATURES" == "true" ] && [ "$TNF_MINIKUBE_ONLY" != "true" ]; then
+
+	export TNF_IMAGE_NAME=cnf-tests
+	export TNF_IMAGE_TAG=latest
+	export TNF_OFFICIAL_ORG=quay.io/openshift-kni/
+	export TNF_OFFICIAL_IMAGE="${TNF_OFFICIAL_ORG}${TNF_IMAGE_NAME}:${TNF_IMAGE_TAG}"
+	export TNF_CMD="/usr/bin/test-run.sh"
+	export OUTPUT_ARG="--junit"
+
+	./run-container.sh -d 127.0.0.53 -n host -o $OUTPUT_LOC -ginkgo.v -ginkgo.skip="performance|sriov|ptp|sctp|xt_u32|dpdk|ovn"
```

Comment:
DNS issue resolved after upgrading my docker engine to the latest version. Will remove the -d argument in this line in the next commit.